objc/objc_runtime_darwin.go

```diff
@@ -215,6 +225,17 @@ func (i Ivar) Offset() uintptr {
 	return ret
 }
 
+// Protocol is a type that declares methods that can be implemented by any class.
+type Protocol uintptr
+
+// GetProtocol returns the protocol for the given name or nil if there is no protocol by that name.
+func GetProtocol(name string) *Protocol {
```

Comment:
Let's comment that this is nil when the protocol is not found.

Reply:
It is, isn't it?

> GetProtocol returns the protocol for the given name or **nil if there is no protocol** by that name.

Reply:
Oh sorry I missed that...